kinesis-video-pic/src/client/src/StreamEvent.c

```diff
@@ -684,20 +724,22 @@ STATUS streamTerminatedEvent(PKinesisVideoStream pKinesisVideoStream, UPLOAD_HAN
                                                  STREAM_STATE_GET_ENDPOINT |
                                                  STREAM_STATE_READY)), retStatus);
 
-    // Stop the stream
-    pKinesisVideoStream->streamState = STREAM_STATE_STOPPED;
+    if (spawnNewUploadSession) {
+        // Stop the stream
+        pKinesisVideoStream->streamState = STREAM_STATE_STOPPED;
 
-    // Get the accepted state
-    CHK_STATUS(getStateMachineState(pKinesisVideoStream->base.pStateMachine, STREAM_STATE_STOPPED, &pState));
+        // Get the accepted state
```

Comment:
Seems that we have \t in the lines here and below

Reply:
this block is inside a if statement